src/main/xerus/monstercat/api/Player.kt

```diff
@@ -164,12 +174,28 @@ object Player: FadingHBox(true, targetHeight = 25) {
 		}
 	}
 	
-	private val pauseButton = ToggleButton().id("play-pause").onClick { if(isSelected) player?.pause() else player?.play() }
-	private val stopButton = buttonWithId("stop") { reset() }
-	private val volumeSlider = Slider(0.0, 1.0, Settings.PLAYERVOLUME()).scrollable(0.05).apply {
-		prefWidth = 100.0
-		valueProperty().listen { updateVolume() }
-	}
+	private val pauseButton = ToggleButton().id("play-pause")
+			.onClick { if (isSelected) player?.pause() else player?.play() }
+			.apply { tooltip = Tooltip("Pause / Play") }
```

Comment:
I have commented on this 3 times by now 🤔

Reply:
This is definitely NOT relevant to this PR. Comment on #15 please.

Reply:
Then please change the target branch to the playlist branch until that is merged, so that I don't see the changes from that.

Reply:
Cannot do that.